src/utils/importFuncGenerator.ts

```diff
@@ -0,0 +1,137 @@
+      options,
+    );
+  }
+
+  const createFuncImport = (...implicits: Implicits[]) =>
+    createImportFuncFuncDefinition(
+      name,
+      path,
+      new Set(implicits),
+      inputs ?? [],
+      outputs ?? [],
+      ast,
+      sourceUnit,
+      options,
+    );
+  const createStructImport = () => createImportStructFuncDefinition(name, path, ast, sourceUnit);
+
+  const warplibFunc = warplibImportInfo.get(path)?.get(name);
+  if (warplibFunc !== undefined) {
+    return createFuncImport(...warplibFunc);
+  }
+
+  switch (path + name) {
+    case STARKWARE_CAIRO_COMMON_ALLOC + 'alloc':
+      return createFuncImport();
+    case STARKWARE_CAIRO_COMMON_BUILTINS + 'BitwiseBuiltin':
+      return createStructImport();
+    case STARKWARE_CAIRO_COMMON_BUILTINS + 'HashBuiltin':
+      return createStructImport();
+    case STARKWARE_CAIRO_COMMON_CAIRO_KECCAK + 'finalize_keccak':
+      return createFuncImport('range_check_ptr', 'bitwise_ptr');
+    case STARKWARE_CAIRO_COMMON_DEFAULT_DICT + 'default_dict_new':
+      return createFuncImport();
+    case STARKWARE_CAIRO_COMMON_DEFAULT_DICT + 'default_dict_finalize':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_DICT + 'dict_read':
+      return createFuncImport('dict_ptr');
+    case STARKWARE_CAIRO_COMMON_DICT + 'dict_write':
+      return createFuncImport('dict_ptr');
+    case STARKWARE_CAIRO_COMMON_DICT_ACCESS + 'DictAccess':
+      return createStructImport();
+    case STARKWARE_CAIRO_COMMON_MATH + 'split_felt':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_MATH_CMP + 'is_le':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_MATH_CMP + 'is_le_felt':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'Uint256':
+      return createStructImport();
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'uint256_add':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'uint256_eq':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'uint256_le':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'uint256_lt':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'uint256_mul':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_CAIRO_COMMON_UINT256 + 'uint256_sub':
+      return createFuncImport('range_check_ptr');
+    case STARKWARE_STARKNET_COMMON_SYSCALLS + 'deploy':
+      return createFuncImport('syscall_ptr');
+    case STARKWARE_STARKNET_COMMON_SYSCALLS + 'emit_event':
+      return createFuncImport('syscall_ptr');
+    case STARKWARE_STARKNET_COMMON_SYSCALLS + 'get_caller_address':
+      return createFuncImport('syscall_ptr');
+    case STARKWARE_STARKNET_COMMON_SYSCALLS + 'get_contract_address':
+      return createFuncImport('syscall_ptr');
```

Comment:
An `if` clause would be more readable here than switch

Reply:
Honestly, I think using switch case here is better. Having to write the if condition all the time would make it more difficult to read

Reply:
I mean `if` could group cases yielding the same result. And there are a lot of those. Alternatively a good option would be object syntax. It's just more concise. Imo it could also be divided into two blocks: struct imports and func imports to further improve readability

Reply:
It's not that I insist on if. Let's try to find the best solution :) There are multiple solutions apart from if, we could use object syntax as I mentioned before, we could also try to deplete the `case` to one liners. What I wonder is if it was the best choice to concatenate path and name. Wouldn't it be simpler to write something like that?
```
if (path === STARKWARE_CAIRO_COMMON_UINT256) {
 assert(UINT256_FUNCTIONS.includes(name), importError(path, name));
 return createFuncImport('range_check_ptr');
}

Reply:
Not sure how it works internally but imo it wouldn't hurt to have either `createFuncImport('range_check_ptr')` as a module constant or a function `rangeCheckPtr = () => createFuncImport('range_check_ptr')`

Reply:
I thought a while and let's leave it as it is. I think that import mechanism requires a broader change anyway. That is bringing warplib structure to js as constants instead of using strings and then generating import strings using the structure. It would overextend the PR scope though